WPMedia/fusion-news-theme-blocks
# added configurations for collections search

**Be sure to run `npm test`, `npm run lint`, and write detailed test steps before requesting reviewers**

## Description

#### Jira Ticket: [THEMES-1633](https://arcpublishing.atlassian.net/browse/THEMES-1633)

Updated the content-api-collections block to allow for searchable collections

## Test Steps

_Add detailed test steps a reviewer must complete to test this PR_

1. Checkout this branch `git checkout THEMES-1633`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/collections-content-source-block`
3. Open a page in Pagebuilder, and add a list block.
4. In each block, under 'Display Content Info' select 'ans-feed', and under 'Content Source' select 'content-api-collections'
5. Verify that the "Select collection" button appears, and select a collection
6. Verify that the configure content section correctly populates the collection id and that the collection is successfully rendered in the block.

<img width="322" alt="image" src="https://github.com/WPMedia/arc-themes-blocks/assets/85515364/f8ea10d3-ffeb-4791-9d88-9a3ff55aa4f9">

[THEMES-1633]: https://arcpublishing.atlassian.net/browse/THEMES-1633?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
